applications/StructuralMechanicsApplication/custom_advanced_constitutive/generic_anisotropic_3d_law.h

```diff
@@ -0,0 +1,481 @@
+// KRATOS  ___|  |                   |                   |
```

Comment:
why in a new directory? ... normally we try not to add new directories, to have the same structure in all of the applications